resources/migrations/001_update_migrations.yaml

```diff
@@ -5228,6 +5228,26 @@ databaseChangeLog:
             tableName: metabase_field
             indexName: idx_field_name_lower
 
+  - changeSet:
+      id: v49.2024-01-26T20:43:18
+      author: qwef
+      comment: >-
+        Added 0.49.0 - Add dashboard_id to cards. Used for when a question is dependent on a dashboard.
+      changes:
+        - addColumn:
+            tableName: report_card
+            columns:
+              - column:
+                  name: dashboard_id
+                  type: int
+                  remarks: "`dashboard_id` will be populated when this card is dependent on a dashboard"
+                  constraints:
+                    nullable: true
```

Comment:
This should definitely be an actual FK. Right now looks like you can set this to `-1000` and no one will complain